cmd/gossamer/main_test.go

```diff
@@ -202,10 +204,7 @@ func TestMain(m *testing.M) {
 		panic(err)
 	}
 
-	defaultKusamaConfigPath = filepath.Join(rootPath, "./chain/kusama/config.toml")
-	defaultPolkadotConfigPath = filepath.Join(rootPath, "./chain/polkadot/config.toml")
-	defaultWestendDevConfigPath = filepath.Join(rootPath, "./chain/westend-dev/config.toml")
-
+	testWestendDevConfigPath = filepath.Join(rootPath, "./chain/westend-dev/config.toml")
```

Comment:
You can't be sure that `TestMain` will run first and `testWestendDevConfigPath` set with the correct file path.  If you ran `TestInitCommand_RenameNodeWhenCalled` it'll come be empty string.

Reply:
I'm not sure I understand correctly, but `TestMain` always runs before any other tests of the package it's in. Printing `testWestendDevConfigPath` at the start of `TestInitCommand_RenameNodeWhenCalled` is not empty either 🤔

Reply:
Oh I didn't see it was `*testing.M`.  I thought it was a test.